spec/support/helpers/services_request_helpers.rb

```diff
@@ -10,14 +10,28 @@ def stub_rummager(content_id)
     allow(Services.rummager).to receive(:search).with(search_params).and_return(rummager_results)
   end
 
+  def stub_rummage_with_doc_type(content_id, document_type)
+    search_params = params(content_id, document_type)
+    allow(Services.rummager).to receive(:search).with(search_params).and_return(rummager_results_same_doc_type)
+  end
+
   private
 
-  def params(content_id)
-    {
-      start: 0,
-      count: RummagerSearch::PAGE_SIZE_TO_GET_EVERYTHING,
-      filter_taxons: [content_id]
-    }
+  def params(content_id, document_type = nil)
```

Comment:
This looks like a duplicate of the `search_params` method in `RummagerSearch`. Is it possible to DRY this up?

Reply:
I removed this method as they are not needed in the new refactor of the code.